bin/vgac2pps.py

```diff
@@ -46,10 +46,9 @@
     parser.add_argument('-all_ch', '--all_channels', action='store_true',
                         help="Save all 21 channels to level1c4pps file.")
     parser.add_argument('-n19', '--as_noaa19',
-                        choices=[version for version in SBAF],
                         default=None,
-                        help=("Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. "
-                              "And apply SBAFs to the channels."))
+                        help=(f"Save only the AVHRR (1,2, 3B, 4, 5) channels to level1c4pps file. "
```

Comment:
f-string should only  be used if you want to include variables, so remove the f from this row.